backend/controllers/accountController.js

```diff
@@ -46,7 +55,6 @@ exports.login = async (req, res) => {
             success: true,
             regCode: regCode,
             isInstructor: isInstructor,
-            accountId: accountId,
```

Comment:
nice! i think this should be good since accountId is always gonna be inferred from the token so good stuff